cmd/celestia-appd/cmd/root.go

```diff
@@ -255,23 +254,22 @@ func NewAppServer(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts se
 }
 
 func createAppAndExport(
-	logger log.Logger, db dbm.DB, traceStore io.Writer, height int64, forZeroHeight bool, jailWhiteList []string,
+	logger log.Logger,
+	db dbm.DB,
+	traceStore io.Writer,
+	height int64,
+	forZeroHeight bool,
+	jailWhiteList []string,
 	appOpts servertypes.AppOptions,
 ) (servertypes.ExportedApp, error) {
-	encCfg := encoding.MakeConfig(app.ModuleEncodingRegisters...) // Ideally, we would reuse the one created by NewRootCmd.
-	encCfg.Codec = codec.NewProtoCodec(encCfg.InterfaceRegistry)
```

Comment:
[blocking]
why is the interface registry no longer registered?

Reply:
it's already registered by the previous line because `encoding.MakeConfig` does it

https://github.com/rootulp/celestia-app/blob/413befd6ade08e38973aa051f3f3814201342853/app/encoding/encoding.go#L41